manage.py

```diff
@@ -0,0 +1,19 @@
+import os
+import sys
+
+
+def main() -> None:
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "taxi_service.settings")
+    try:
+        from django.core.management import execute_from_command_line
+    except ImportError as exc:
+        raise ImportError(
+            "Couldn't import Django. Make sure you installed it and "
+            "Make sure you're PYTHONPATH have correct variables."
+            "Make sure you activate venv"

```

Comment:
The error message could be clearer. Consider rephrasing it to: 'Couldn't import Django. Ensure it's installed and available on your PYTHONPATH environment variable. Also, make sure your virtual environment is activated.' This will make it more understandable for users encountering the ImportError.